index.bs

```diff
@@ -8,7 +8,7 @@ Abstract: This document extends the API in [[FS]] to enables developers to build
 Status: CG-DRAFT
 ED: https://wicg.github.io/file-system-access/
 Level: 1
-Editor: Marijn Kruisselbrink, Google, mek@chromium.org, w3cid 72440
+Editor: Austin Sullivan, Google, asully@chromium.org, w3cid 126126
```

Comment:
Presumably your w3cid is `https://www.w3.org/users/<wc3id>` corresponding to your profile?